@navikt/core/css/accordion.css

```diff
@@ -20,16 +25,50 @@
   background: var(--navds-accordion-color-background);
 }
 
+.navds-accordion__header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: var(--navds-spacing-4);
+  padding: var(--navds-spacing-4) var(--navds-spacing-3);
+  margin: 0;
+  text-align: left;
+  background: transparent;
+  cursor: pointer;
+  border: none;
+  border-bottom: 2px solid var(--navds-accordion-color-border);
+}
+
 .navds-accordion__header:focus {
   outline: none;
   box-shadow: var(--navds-shadow-focus);
-  border-color: var(--navds-accordion-color-border-focus);
 }
 
 .navds-accordion__header:hover {
   color: var(--navds-accordion-color-text-hover);
   border-color: var(--navds-accordion-color-border-hover);
-  text-decoration: underline;
+}
+
+.navds-accordion__header--open {
```

Comment:
For meg gir `.navds-accordion__item--open > .navds-accordion` mer mening her fordi det er accordion item som er åpen, ikke selve headeren.

Reply:
Enig, fikset det 🆗